teabot.service.template

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Teabot - tea reporting daemon.
+After=network.target
+
+[Service]
+Type=simple
+User=root
+ExecStart={{PWD}}/run {{ARGS}}
+Restart=always
```

Comment:
Ah so SystemD also does process supervision, that's nice, means we don't need supervised now